AdobeStockImageAdminUi/view/adminhtml/web/js/mediaGallery/grid/columns/image/licenseActions.js

```diff
@@ -98,6 +98,7 @@ define([
                     this.imageModel().addMessage('success', $.mage.__('The image has been licensed.'));
                 }.bind(this)).fail(function (error) {
                     if (error) {
+                        $('ul.action-menu').removeClass('_active');
```

Comment:
The context menu mentioned in an issue is a part of Magneto media gallery implementation. Adobe Stock Integration should not interact directly with elements of the media gallery. Also, I think even for media gallery components it would be better to interact with `collapsible` component rather then with it's HTML elements.

I'd ensure the collapsible is closed in `Magento_MediaGalleryUi/js/grid/columns/image.js::addMessage` method